src/CommunityToolkit/Xamarin.CommunityToolkit.Sample/Xamarin.CommunityToolkit.Sample.csproj

```diff
@@ -40,4 +30,17 @@
   <ItemGroup>
     <ProjectReference Include="..\Xamarin.CommunityToolkit\Xamarin.CommunityToolkit.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
please keep AppResources together

Reply:
If you see the `main` branch, we keep one empty line between `ItemGroups` tags

Reply:
And after we merge this PR we can discuss betters ways to organize our csproj. I don't want to change anything else here, to avoid merge conflicts in the future. Since I don't know how many PR's we will merge on `main`.